### DISCUSSION TITLE
HDMI video and audio

### DISCUSSION THREAD

**Discussion by Hobbyist11** [thesofproject/sof#9395] [Q&A] [answered]
Greetings, I'm experiencing really bad flickering (turning on and off) of an HDMI external display on an Intel Meteorlake machine. I've noticed that it only happens when audio is set to play on HDMI but not when it is set to play on laptop.  I've filed an issue to i915. https://gitlab.freedesktop.org/drm/i915/kernel/-/issues/11391#note_2476569 but debugging it, haven't found any possible causes/clues. Could this be an SOF issue? I've also enabled verbose debugging for SOF but haven't found anything useful too, nothing appears on dmesg -w when the flickering happens. Uninstalling sof-firmware though makes the flicker disappear at the expense of not having audio.

**Comment by plbossart** [comment, marked as answer]
moved to issue https://github.com/thesofproject/linux/issues/5153
